repo-agent/review-ui/review-api/main.go

```diff
@@ -1859,3 +1901,157 @@ func proxy(c *gin.Context) {
 
 	c.String(resp.StatusCode, string(body))
 }
+
+// TODO this is k8s specific. we need to generalize it later.
+var allowedLabelPrefixes = []string{"area/", "kind/", "priority/", "sig/", "type/"}
+
+func getSuggestedLabels(ctx context.Context, client *github.Client, owner, _, repo string) ([][]string, error) {
```

Comment:
nit: user isn't used in the call here w/ @me used so it might make sense to remove from the signature

Reply:
removed.